pkg/registry/apis/secret/accesscontrol.go

```diff
@@ -0,0 +1,116 @@
+package secret
+
+import (
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+const (
+	// SecureValues
+	ActionSecretSecureValuesCreate = "secret.securevalues:create" // CREATE.
+	ActionSecretSecureValuesWrite  = "secret.securevalues:write"  // CREATE + UPDATE.
```

Comment:
nit: inconsistent with the keepers annotations below, is this intentional?

Reply:
typo 😄